app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -240,6 +242,11 @@ object TestHelper {
         }
     }
 
+    fun getRunningTask(task: String) {
```

Comment:
`getRunningTask` indicates this method will return a "Task" but it actually receives a `task: String` argument so it is a bit confusing.

Maybe this method could be refactored to become `getTopActivityInCurrentTask()` which would return `activityManager.appTasks[0].taskInfo.topActivity`?
Or `isShowingExternalContent()` which would return a Boolean after checking
```kotlin
activityManager.appTasks[0].taskInfo.topActivity!!.className == ExternalAppBrowserActivity::class.java.name
```

Reply:
Sorry for the confusion, took your advice and created `isExternalAppBrowserActivityInCurrentTask` which would return a Boolean.